Development/nmos/json_fields.h

```diff
@@ -230,6 +230,114 @@ namespace nmos
         const web::json::field_as_string hostname{ U("hostname") }; // hostname, ipv4 or ipv6
         const web::json::field_as_integer port{ U("port") }; // 1..65535
 
+        // IS-12 Control Protocol
+        namespace nc
```

Comment:
Hmmmm,  Not sure what the right thing to do here is with the namespace name:

`ncp` is what's used in the IS-12 spec to describe the protocol.  However, this namespace also includes the MS-05 field definitions.

So do we state that the `nc` namespace covers IS-12, MS-05 and in future IS-14?
Or do we segment the namespace further?

I would lean towards stating that `nc` covers both IS-12 and MS-05 perhaps...
```suggestion
        // IS-12 Control Protocol and MS-05 model definitions
        namespace nc
```